### PR TITLE
Bulk import practitioner roles, practitioners and organizations 

### PR DESCRIPTION
**Pull request by ellykits**

### CODE REVIEW

File: src/main/java/org/opensrp/web/rest/PractitionerResource.java [hunk 1]
```diff
@@ -130,4 +131,32 @@ private PractitionerSearchBean createPractitionerSearchBean(Integer pageNumber,
...
+            else
+                return new ResponseEntity<>("Practitioners Ids not processed: " + String.join(",", unprocessedIds),
+                        HttpStatus.CREATED);
+
+        }
+        catch (JsonSyntaxException e) {
+            logger.error("The request doesnt contain a valid practitioner representation",e);
+            return new ResponseEntity<>(HttpStatus.BAD_REQUEST);
+        }
+    }
```
Comment (githengi) [inline src/main/java/org/opensrp/web/rest/PractitionerResource.java:161]: This is not required and should be handled by Global Exception Handler

File: src/main/java/org/opensrp/web/rest/OrganizationResource.java [hunk 2]
```diff
@@ -50,8 +38,14 @@
 import org.springframework.web.bind.annotation.RequestMethod;
 import org.springframework.web.bind.annotation.RequestParam;
 
-import com.fasterxml.jackson.core.JsonProcessingException;
-import com.fasterxml.jackson.databind.ObjectMapper;
+import java.util.*;
```
Comment (Rkareko) [inline src/main/java/org/opensrp/web/rest/OrganizationResource.java:41]: Let's avoid wild card imports

File: src/main/java/org/opensrp/web/rest/OrganizationResource.java [hunk 3]
```diff
@@ -1,30 +1,17 @@
 /**
- * 
+ *
  */
 package org.opensrp.web.rest;
 
-import static org.opensrp.web.Constants.ORDER_BY_FIELD_NAME;
```
Comment (Rkareko) [inline src/main/java/org/opensrp/web/rest/OrganizationResource.java:6]: Are we using the same formatter?

File: src/main/java/org/opensrp/web/rest/OrganizationResource.java (hunk 2 above)
Comment (ellykits) [reply src/main/java/org/opensrp/web/rest/OrganizationResource.java:41]: Replacing this with qualified imports

File: src/main/java/org/opensrp/web/rest/OrganizationResource.java (hunk 3 above)
Comment (ellykits) [reply src/main/java/org/opensrp/web/rest/OrganizationResource.java:6]: Yes using OpenMRS formatter

File: src/test/java/org/opensrp/web/rest/OrganizationResourceTest.java [hunk 4]
```diff
@@ -40,17 +17,12 @@
 import org.keycloak.KeycloakSecurityContext;
 import org.keycloak.adapters.RefreshableKeycloakSecurityContext;
 import org.keycloak.representations.AccessToken;
-import org.mockito.ArgumentCaptor;
-import org.mockito.ArgumentMatchers;
-import org.mockito.Captor;
-import org.mockito.Mock;
-import org.mockito.MockitoAnnotations;
+import org.mockito.*;
```
Comment (Rkareko) [inline src/test/java/org/opensrp/web/rest/OrganizationResourceTest.java:20]: This class has multiple wild card imports

File: src/test/java/org/opensrp/web/rest/PractitionerResourceTest.java [hunk 5]
```diff
@@ -17,16 +16,10 @@
 import java.util.List;
 import java.util.Set;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.*;
```
Comment (Rkareko) [inline src/test/java/org/opensrp/web/rest/PractitionerResourceTest.java:19]: Let's avoid wilcard imports

**Review by Rkareko — APPROVED** [review]

**Pull request merged**